### PR TITLE
Ported to TypeScript

### PR DESCRIPTION
**Pull request by kolatat**
- as of #3

### CODE REVIEW

**Comment by kolatat** [comment]
Need you to test cuz I dont have token

**Comment by aa-tan** [comment]
can I just `node bot.ts` or do I need compilation tool

**Comment by kolatat** [comment]
You will need to compile (see package.json postinstall script) `npm run postinstall` === `tsc --project .`. The tsc compilation tool will already be installed as it is a dependency (typescript).

When in doubt, `npm install` which will both install dependencies and do the postinstall compilation.

You can also make your IDE/Editor watch changes to \*.ts files and autocompile them into \*.js. Autocompilation is already enabled in IntelliJ settings, and VSCode should have native support? or just have to enable it first.

File: .gitignore [hunk 1]
```diff
@@ -61,3 +61,73 @@ typings/
...
+fabric.properties
+
+# Editor-based Rest Client
+.idea/httpRequests
+
+# Android studio 3.1+ serialized cache file
+.idea/caches/build_file_checksums.ser
+
+**/*.js
+**/*.js.map
```
Comment (aa-tan) [inline .gitignore]:
```suggestion
**/*.js.map

# Project Build location
build/
```
Add build location for compiled JS?

File: package.json [hunk 2]
```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": "bot.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "tslint --project .",
+    "postinstall": "tsc --project .",
```
Comment (aa-tan) [inline package.json]:
```suggestion
    "postinstall": "tsc --project . --outdir ./build",
```

Add a build directory to make things clean

**Review by aa-tan — CHANGES_REQUESTED** [review]
Make those changes and I think it'll be good to merge. Also, this suggested change beta thing is pree cool

**Pull request merged**